sprint0/sprint0/Weapons/Arrow.cs

```diff
@@ -33,6 +34,7 @@ public Arrow(Texture2D texture, Vector2 location, Direction dir, IEntity shooter
                 tipOffset = width * DirectionMethods.ToVector2(dir) + height * DirectionMethods.ToVector2(dir);
 
             Location = new Rectangle((int)loc.X, (int)loc.Y, (int)(width * Game1.Scale), (int)(height * Game1.Scale));
+            LocationDraw = new Rectangle((int)loc.X, (int)loc.Y, (int)(width * Game1.Scale), (int)(height * Game1.Scale));
```

Comment:
is this supposed to have slightly different values? would it would be helpful to just crop arrow if transparent pixels are causing the bigger hitbox?

Reply:
it's not transparent pixels, its when the arrow gets rotated to shoot east or west, the hitbox the collision detect uses doesn't get rotated with it